examples/meta/src/neural_nets/classification_perceptron.sg

```diff
@@ -0,0 +1,35 @@
+CSVFile f_feats_train("../../data/classifier_binary_2d_linear_features_train.dat")
```

Comment:
ah, I had overseen this.
Can you pls send another PR where you  move the perceptron to the "binary" folder, it does not really follow the NN structure in Shogun. Same for the data file.
Also could you remove the python example in the same PR?

Reply:
Ahh ok, I thought it had to be with NN examples, I'll move it and remove the python example...

Also... I'm seeing that one check didn't pass which is weird because I just send the data file... don't know :/ 

Well, I'lll do that and send another commit!

Reply:
another PR* xp